### ISSUE TITLE
Create IFR view

### ISSUE THREAD

**Issue by gilbertohasnofb** [c172p-team/c172p-detailed#880]
Create IFR view:

![fgfs-screen-001](https://cloud.githubusercontent.com/assets/5700795/20684060/b713c88e-b5ae-11e6-91ec-9d604c35bedb.png)

**Comment by gilbertohasnofb** [comment]
What do you guys think about this? Add an extra view so that if the user wants to fly fully IFR s/he can simply use that view instead of having to set it on their own.

**Comment by onox** [comment]
What do you mean exactly? Do you just want to a have a view (camera) that is more in the center and lower? I don't think any pilot in RL is gonna fly like that. 😮

**Comment by onox** [comment]
But perhaps we can do the following: sometimes pilots fly with special glasses so that they cannot look outside (only at instruments in cockpit) (instructor still can). Don't know if we can use ALS glass effect for that.

**Comment by gilbertohasnofb** [comment]
> What do you mean exactly? Do you just want to a have a view (camera) that is more in the center and lower? 

Yep :smile:

> I don't think any pilot in RL is gonna fly like that. :open_mouth:

Surely no pilot would fly from that angle, in RL they use this for IFR training:

![](http://www.langleyflyingschool.com/Photos/People%20and%20Places/2009/Sukhmani%20009.jpg)

But having a view of the panel from that angle I proposed is quite a common feature in other sims as an aid for training: it's not suppose to be a realistic physical point of view, it's just there to make you not see the exterior and force you to fly solely IFR. Sims are also training tools, and I see no problem in adding such a feature for those who want to train. As for using ALS to simulate the blindfold, that could be an idea, but it would be more complicated and IMO it would actually be worse: the point of IFR training in a sim is not to make you feel how would be to wear a blindfold, but how it would be to only fly with instruments. Having all the instruments, switches and toggles in the screen at the same time without having to look around frees you to solely learn IFR instead of worrying about other things. That's my 2 cents at least.

**Comment by onox** [comment]
> As for using ALS to simulate the blindfold, that could be an idea, but it would be more complicated and IMO it would actually be worse: the point of IFR training in a sim is not to make you feel how would be to wear a blindfold, but how it would be to only fly with instruments.

It's not like you're completely blindfolded. You can still the instruments, but you cannot see through the windows because of some special coating / foil on the glass.

But your solution is much easier to achieve, so let's go with that.

**Comment by gilbertohasnofb** [comment]
> It's not like you're completely blindfolded. You can still the instruments, but you cannot see through the windows because of some special coating / foil on the glass.

I never seen that before! I am only familiar with the blindfold helmet

> But your solution is much easier to achieve, so let's go with that.

:+1:

**Comment by gilbertohasnofb** [comment]
Reminder to myself: test view on both 4:3 and 16:9 ratios of resolution

**Comment by onox** [comment]
Yeah, I cannot really find any references to it anymore via Google. Google only shows those blindfold helmets or glasses, which cover part of the view.

**Comment by wlbragg** [comment]
I'm not necessarily suggesting not using @gilbertohasnofb approach. But it would be really cool to have a GUI choice to use IFR training visor/helmet. Wouldn't a visor model placed in front of the pilot view actually be very simple to make and select animate through a GUI choice?
I'll volunteer to make the visor/helmet, or maybe @dg-505 would like to try practice his Blender skills?

**Comment by dg-505** [comment]
Having such a visor would be really a unique feature of this aircraft 😆 

I can give it a try, but it will take some time because I'm planning to update the titorials next. So Wayne, feel free to start modelling :wink:

**Comment by gilbertohasnofb** [comment]
Well, let me put it this way: a visor is just a tool you use to accomplish the same as I propose. Simulating a visor might seem like the "real" approach, but I think we are missing some important points here. Flying in a sim is definitely not the same as flying in RL. In RL, you are constantly moving your point of view (please check the head movements of this 172 pilot here https://www.youtube.com/watch?v=YeNOHeL127Y ) while in the sim we are almost stuck to the default viewpoint due to the lack of controls (unless you use a VR or something like that, which I don't even think FG supports). I really think that when wearing a blindfold visor, one has to make even more movements than normal just to be able to check the instruments, is that really feasible in a sim? I see us implementing this feature, everyone goes "cool" and nobody ever uses it for IFR training because it's too impractical. 

I once had the opportunity to try out a simulator for the EMB 312 Tucano from the Brazilian Air Force, and it looks like this:

![](https://lh4.googleusercontent.com/-BLlFZ32XtE4/U-tkW2NXgyI/AAAAAAAALT4/m-wHb_TO7J0/w504-h664-no/AT3T0592.JPG)

Notice that there are no screens or monitors or projectors, as that is a IFR _only_ training simulator. Is that an unrealistic approach? Certainly not from the point of view of training, perhaps from the point of view of simulating what is to be in the air in a Tucano. It depends on what you are trying to accomplish I suppose.

TL;DR If you guys insist, go ahead and implement this, but I think my proposed approach is not only a million times simpler, but a million times more effective as a training tool.

**Comment by mflas** [comment]
My vote would be to add a view like @gilbertohasnofb proposed. It's what I use for instrument flying in Other Simulators[tm]. Usability wise I really prefer that approach to some kind of foggles / blocking windows. When flying IFR in the simulator, you really need most of the monitor real estate to go to rendering instruments. It's much more important in IFR flying to be able to precisely read everything, compared to VFR where you ought to mostly be looking outside and just glance down on your instruments.

Another point is that in the sim you can easily adjust the weather to be as low-viz as you want, so you don't need a hood or foggles to simulate bad weather.

**Comment by algefaen** [comment]
Bah, that was me commenting from the wrong account. (=

**Comment by wlbragg** [comment]
>If you guys insist, go ahead and implement this,

No insisting here. Continue with your idea @gilbertohasnofb. Maybe down the road if I feel energetic I might attempt to make a visor/helmet as shown in the picture just to see if it would work. "IF" it did, and worked well, it would be a unique accessory.

**Comment by gilbertohasnofb** [comment]
In that case, I will go ahead and implement this soon and then you can think about the visor in the future, is that fine?
